ci/infra/testrunner/tests/test_cilium.py

```diff
@@ -52,3 +52,34 @@ def test_cillium(deployment, kubectl):
         time.sleep(30)
 
     assert all_reachable
+
+
+def test_cilium_l7(deployment, kubectl):
+    """
+    GIVEN cilium is properly configured
```

Comment:
this sounds like it wants to be BDD but then its a normal test. Are we gonna introduce BDD tests?

Reply:
We could. The above is meant to just be the test case in plane english.

Reply:
:+1: thanks for the clarification!